cmd/argocd/commands/app.go

```diff
@@ -2518,10 +2518,12 @@ func checkResourceStatus(watch watchOpts, healthStatus string, syncStatus string
 		return false
 	}
 
+	// If resource does not have health status, we default to healthy.
```

Comment:
```suggestion
	// If resource does not have health status, we default to healthy.
```
nit; i think this comment makes more sense at [line 2524](https://github.com/argoproj/argo-cd/pull/23170/files#diff-95f228706705fc556fbb63b303d638ad4d4ddba8118a50dc83e344ac9b692d49R2524)